source/documentBase.py

```diff
@@ -169,6 +165,7 @@ def _tableMovementScriptHelper(self, movement="next", axis=None):
 		speech.speakTextInfo(info,formatConfig=formatConfig,reason=controlTypes.REASON_CARET)
 		info.collapse()
 		self.selection = info
+		#info.updateCaret()

```

Comment:
remove this or add some comments about why we specifically don't want to do this here.